include/libdnf/rpm/package_query.hpp

```diff
@@ -611,6 +611,9 @@ class PackageQuery : public PackageSet {
     /// @param timestamp Only packages built after this will pass
     void filter_recent(const time_t timestamp);
 
+    /// Keep in the query only user-installed packages.
```

Comment:
My note will sound not logic but I would suggest to add explicitly that it keeps installed packages. I remember some discussion related to reason and what reason is supposed to be returned for available packages, therefore I would prefer to be explicit here.

What about:
`/// Keep in the query only installed packages that are user-installed.`